---
layout: default
title: "BIO311: Molecular Biology"
course_description: "A detailed exploration of the structure and function of DNA and RNA and how these nucleic acids are used to form proteins, and their importance in genetics and inheritance. Special emphasis is placed on DNA replication, transcription, gene expression, mutation and repair, recombination, molecular techniques, and appropriate molecular models."
next: ../Unit06
previous: ../Unit04
---
**Unit 5: Transcription** <span id="5"></span> 
*Transcription leads to the synthesis of a complementary RNA strand from
a DNA template.  The process is less precise than DNA replication, since
RNA polymerases do not have proofreading activity.  There are remarkable
differences between prokaryotic and eukaryotic transcription, which we
will discuss in detail in Unit 7 (Regulation of Gene Expression).  The
turnover of RNA molecules is commonly regulated by P-bodies and miRNA in
the cell.  This unit focuses on mRNA and miRNA; tRNA and rRNA are
discussed in the next unit.      *

**Unit 5 Time Advisory**  
This unit should take you approximately 12.5 hours to complete.  
  
 ☐    Subunit 5.1: 1.5 hours  
  
 ☐    Subunit 5.2: 0.5 hours  
  
 ☐    Subunit 5.3: 1.5 hours  
  
 ☐    Subunit 5.4: 1.5 hours  
  
 ☐    Subunit 5.5: 4.0 hours  
  
 ☐    Subunit 5.6: 1.5 hour  
  
 ☐    Subunit 5.7: 2.0 hour

**Unit5 Learning Outcomes**  
Upon successful completion of this unit, students will be able to:
-   Describe the three stages of the transcriptional cycle.
-   Compare and contrast translation in prokaryotes and eukaryotes.
-   Compare the characteristics of RNA polymerases.
-   Compare and contrast RNA polymerases and DNA polymerases.
-   Discuss the consequence of RNA splicing and editing.
-   Predict the characteristics of differentially spliced RNA molecules.
-   Describe miRNA-regulated RNA turnover.

**5.1 Basic Transcriptional Cycle** <span id="5.1"></span> 
-   **Lecture: YouTube: Oregon State University: Kevin Ahern's
    "Transcription II"**
    Link: YouTube: Oregon State University: Kevin Ahern's
    "[Transcription II](http://www.youtube.com/watch?v=9J6m4_WBgcc)"
    (YouTube)  
        
     Instruction: Please watch this video (51 min).  
        
     Terms of Use: Please respect the copyright and terms of use
    displayed on the webpage above.

-   **Reading: NCBI Bookshelf: Berg et al.'s "DNA Replication of Both
    Strands Proceeds Rapidly from Specific Start Sites"**
    Link: NCBI Bookshelf: Berg et al.'s "[DNA Replication of Both
    Strands Proceeds Rapidly from Specific Start
    Sites](http://www.ncbi.nlm.nih.gov/books/NBK22587/)" (HTML)  
        
     Instructions:  Please study the following sections: "An RNA Primer
    Synthesized by Primase Enables DNA Synthesis to Begin;" "One Strand
    of DNA Is Made Continuously, Whereas the Other Strand Is Synthesized
    in Fragments;" "DNA Ligase Joins Ends of DNA in Duplex Regions;"
    "DNA Replication Requires Highly Processive Polymerases;" "The
    Leading and Lagging Strands Are Synthesized in a Coordinated
    Fashion;" and "DNA Synthesis Is More Complex in Eukaryotes Than in
    Prokaryotes."  
        
     Terms of Use: Please respect the copyright and terms of use
    displayed on the webpage above.

**5.2 Types of RNA Polymerase** <span id="5.2"></span> 
-   **Reading: NCBI Bookself: Lodish et al.'s "Eukaryotic Gene Control:
    Purposes and General Principles"**
    Link:  NCBI Bookself:  Lodish et al.'s "[Eukaryotic Gene Control:
    Purposes and General
    Principles](http://www.ncbi.nlm.nih.gov/books/NBK21635/)" (HTML)  
        
     Instruction:  Please study the introduction and "Three Eukaryotic
    Polymerases Catalyze Formation of Different RNAs" sections on this
    page.  Please note that these three RNA polymerases have different
    specificity, thus they cannot replace each other.  
        
     Terms of Use: Please respect the copyright and terms of use
    displayed on the webpage above.

**5.3 Transcription in Prokaryotes** <span id="5.3"></span> 
-   **Reading: NCBI Bookshelf: Brown's "Synthesis and Processing of
    RNA"**
    Link: NCBI Bookshelf:  Brown's "[Synthesis and Processing of
    RNA](http://www.ncbi.nlm.nih.gov/books/NBK21132/)" (HTML)  
        
     Instructions:  Please study the "Synthesis of Bacterial mRNAs"
    section on this page.  Translation and transcription take place
    simultaneously in prokaryotes.  Please note that prokaryotic mRNAs
    are not processed.  
        
     Terms of Use: Please respect the copyright and terms of use
    displayed on the webpage above.

**5.4 Transcription in Eukaryotes** <span id="5.4"></span> 
-   **Reading: NCBI Bookshelf: Brown's "Synthesis and Processing of
    RNA"**
    Link: NCBI Bookshelf:  Brown's "[Synthesis and Processing of
    RNA](http://www.ncbi.nlm.nih.gov/books/NBK21132/)" (HTML)  
        
     Instructions:  Please study the "Synthesis of Eukaryotic mRNAs by
    RNA Polymerase II" section on this page, including the "Capping of
    RNA Polymerase II Transcripts Occurs Immediately after Initiation,"
    "Elongation of Eukaryotic mRNAs," and "Termination of mRNA Synthesis
    Is Combined with Polyadenylation" subsections.  In eukaryotes,
    transcription follows translation, and transcription and translation
    take place in different subcellular compartments.  Please note that
    eukaryotic mRNAs are processed with a 5'-cap and 3'-tail.  Capping
    and polyadenylation are post-translational modifications.  
        
     Terms of Use: Please respect the copyright and terms of use
    displayed on the webpage above.

**5.5 RNA processing** <span id="5.5"></span> 
**5.5.1 RNA Splicing** <span id="5.5.1"></span> 
-   **Reading: NCBI Bookshelf: Brown's "Synthesis and Processing of
    RNA"**
    Link: NCBI Bookshelf:  Brown's "[Synthesis and Processing of
    RNA](http://www.ncbi.nlm.nih.gov/books/NBK21132/)" (HTML)  
        
     Instructions:  Please study the "Intron Splicing" section on this
    page, including the following subsections: "Conserved Sequence
    Motifs Indicate the Key Sites in GU-AG Introns," "Outline of the
    Splicing Pathway for GU-AG Introns," "snRNAs and Their Associated
    Proteins Are the Central Components of the Splicing Apparatus,"
    "Alternative Splicing Is Common in Many Eukaryotes," and "AU-AC
    Introns Are Similar to GU-AG Introns but Require a Different
    Splicing Apparatus."  Differential splicing results in multiple RNA
    molecules from one gene.  In differentially spliced RNAs, different
    exon combinations are spliced together.  Exon(s) may correspond to
    protein domains, thus the characteristics of the encoded protein
    become predictable from the combination of the functional domains.  
        
     Terms of Use: Please respect the copyright and terms of use
    displayed on the webpage above.

-   **Reading: Scitable: Dr. Suzanne Clancy's "RNA Splicing: Introns,
    Exons and Spliceosome"**
    Link: Scitable:  Dr. Suzanne Clancy's "[RNA Splicing: Introns, Exons
    and
    Spliceosome](http://www.nature.com/scitable/topicpage/rna-splicing-introns-exons-and-spliceosome-12375)"
    (HTML)  
        
     Instructions:  Please study this page.  
        
     Terms of Use: Please respect the copyright and terms of use
    displayed on the webpage above.

**5.5.2 Trans-Splicing** <span id="5.5.2"></span> 
-   **Reading: NCBI Bookshelf: Riddle et al.'s "Trans-Splicing"**
    Link: NCBI Bookshelf:  Riddle et al.'s
    "[Trans-Splicing](http://www.ncbi.nlm.nih.gov/books/NBK20087/)"
    (HTML)  
        
     Instructions:  Please study this page.  
        
     Terms of Use: Please respect the copyright and terms of use
    displayed on the webpage above.

**5.5.3 RNA Editing** <span id="5.5.3"></span> 
-   **Reading: John W. Kimball’s "RNA Editing"**
    Link:  John W. Kimball’s "[RNA
    Editing](http://users.rcn.com/jkimball.ma.ultranet/BiologyPages/R/RNA_Editing.html)"
    (HTML)  
        
     Instruction:  Please study this page.  
        
     Terms of Use: Please respect the copyright and terms of use
    displayed on the webpage above. 

**5.6 Riboswitches** <span id="5.6"></span> 
-   **Reading: University of Colorado: Garst & Batey's "A Switch in
    Time: Detailing the Life of a Riboswitch"**
    Link:  University of Colorado: Garst & Batey's "[A Switch in Time:
    Detailing the Life of a
    Riboswitch](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2783387/?tool=pubmed)"
    (HTML)  
        
     Instruction:  Please study this page, including the figures
    embedded into the publication.  Please note that binding of
    metabolites to riboswitches  regulates the expression of the mRNA
    wherein the riboswitch is located.  
        
     Terms of Use: Please respect the copyright and terms of use
    displayed on the webpage above.

**5.7 RNA Degradation** <span id="5.7"></span> 
**5.7.1 MicroRNA (miRNA)** <span id="5.7.1"></span> 
-   **Reading: Scitable: Suzanne Clancy's "RNA Functions"**
    Link:  Scitable: Suzanne Clancy's "[RNA
    Functions](http://www.nature.com/scitable/topicpage/rna-functions-352)"
    (HTML)  
        
     Instructions:  Please study this page.   
        
     Terms of Use: Please respect the copyright and terms of use
    displayed on the webpage above.

**5.7.2 P-Bodies** <span id="5.7.2"></span> 
-   **Lecture: YouTube: University of Arizona: Dr. Roy Parker's
    "P-bodies and the mRNA Cycle"**
    Link:  University of Arizona: Dr. Roy Parker's "[P-bodies and the
    mRNA Cycle](http://www.youtube.com/watch?v=THx5SgSpk6w)" (YouTube)  
        
     Instructions: Please watch this video (35 min).  Please note that
    P-bodies serve as a scaffolding center for miRNA, and
    translationally repressed mRNAs are also stored in P-bodies.      
        
     Terms of Use: Please respect the copyright and terms of use
    displayed on the webpage above.


